core/model/edition.py

```diff
@@ -724,6 +724,17 @@ def calculate_author(self):
             sort_author = " ; ".join(sorted(sort_names))
         else:
             sort_author = self.UNKNOWN_AUTHOR
+
+        def truncate_string(mystr: str):
+            if len(mystr) > 1024:
```

Comment:
How did we come to the value `1024` here. The error message mentions `2704`.

```
This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (
raised as a result of Query-invoked autoflush; consider using a session.no_autoflush block if this flush is occurring prematurely)
(psycopg2.errors.ProgramLimitExceeded) index row size 3208 exceeds btree version 4 maximum 2704 for index "ix_editions_author"
```

Reply:
see comment below.